### ISSUE TITLE
Internal server error when creating new version

### ISSUE THREAD

**Issue by drronnie85** [wit-ai/wit#2100]
BUG

**What is the current behavior?**
AppId: 272641183996000
I try to create new version, but request got 500 status code.
After page refresh I see new version, but when I add new tag to request Wit.Ai return no intent

**Comment by chessai** [comment]
See https://github.com/wit-ai/wit/issues/2105#issuecomment-856296095

**Comment by chessai** [comment]
This should be resolved. See https://github.com/wit-ai/wit/issues/2105#issuecomment-860916172

**Comment by drronnie85** [comment]
@chessai it is not resolved(

**Comment by drronnie85** [comment]
@chessai hello. Do you have any information?
Is it problem affects only my application or it general problem?
Thank you

**Comment by drronnie85** [comment]
@chessai I think I found problem.
Steps to reproduce:
1. Create new intent with entity (in my case wit\duration)
2. Wait training complete
3. Remove intent.
4. Check entity - it exists but not related to any intents
5. Wait training complete
6. Create tag produce this error

I reproduced this bug only one time.
In 272641183996000 app I resolved problem by removing entity that not related to any intent.
